src/components/admin/dashboards/dashboard/toolbar.jsx

```diff
@@ -78,7 +78,10 @@ const Toolbar = ({
     setData(keep);
     api({
       method: "DELETE",
-      url: `/api/dashboard/${page}?remove=${ids.join(",")}`,
+      url: `/api/dashboard/${page}`,
+      body: {
```

Comment:
we can pass in rows directly, also rows is prolly a v large data structure, lets only pass what we need to